foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -555,14 +557,14 @@ msgid ""
 "password. You can then set up your mobile devices, using the QR code "
 "available next to the form."
 msgstr ""
-"Wenn Sie den Router als Wi-Fi-Zugangspunkt verwenden wollen, dann aktivieren "
-"Sie die Wi-Fi-Option. Geben Sie dann die SSID (Netzwerkname) und ein "
+"Wenn Sie den Router als WLAN-Zugangspunkt verwenden wollen, dann aktivieren "
+"Sie die WLAN-Option. Geben Sie dann die SSID (Netzwerkname) und ein "
 "dazugehöriges Kennwort ein.<br><br>Danach können Sie Ihre Mobilgeräte "
 "mittels des angezeigten QR-Codes einfach konfigurieren."
 
 #, python-format
 msgid "Enable Wi-Fi %s"
-msgstr "Aktiviere Wi-Fi %s"
+msgstr "Aktiviere WLAN %s"
```

Comment:
I'm willing to accept WLAN. I've just repeated my comment because I was not sure about it. I should trust you as you are the native speaker.